discovery_engine_core/core/src/storage/sqlite.rs

```diff
@@ -91,7 +96,40 @@ impl SqliteStorage {
         Ok(Self { pool })
     }
 
-    async fn begin_tx(&self) -> Result<Transaction<'_, Sqlite>, Error> {
+    async fn setup_keep_stacks_in_sync(tx: &mut Transaction<'_, Sqlite>) -> Result<(), Error> {
+        let expected_ids = &[
```

Comment:
🔧 since all values are constant, we can also make the slice constant
```suggestion
        const EXPECTED_IDS: &[stack::Id] = &[
```

Reply:
In the best case this will  be turned to exactly the same code as I'm using and in the worst case would turn it into a `static` which permanently would increase the runtime memory.

There is AFIK no drawback of having static stack values in "cold" code.